internal/storage/minio_chunk_manager.go

```diff
@@ -312,3 +319,19 @@ func (mcm *MinioChunkManager) ListWithPrefix(prefix string, recursive bool) ([]s
 	}
 	return objectsKeys, nil
 }
+
+func (mcm *MinioChunkManager) GetModTime(filename string) (time.Time, error) {
+	object, err := mcm.Client.GetObject(mcm.ctx, mcm.bucketName, filename, minio.GetObjectOptions{})
+	if err != nil {
+		log.Error("minio getObject error", zap.String("bucketName", mcm.bucketName), zap.String("filename", filename))
+		return time.Now(), err
+	}
+
+	objectInfo, err := object.Stat()
+	if err != nil {
+		log.Error("minio stat error", zap.String("bucketName", mcm.bucketName), zap.String("filename", filename))
+		return time.Now(), err
```

Comment:
you could just return time.Time{} (0001-01-01 00:00:00 +0000 UTC) instead of time.now(). and time.Time.IsZero(time.Time{}) == true

Reply:
ok

Reply:
ok